hugolib/node_as_page_test.go

```diff
@@ -385,6 +386,25 @@ title = "Deutsche Hugo"
 	assertFileContent(t, expectedFilePath(ugly, "public", "nn", "sect1", "regular2"), true,
 		"Single Title: Page 02")
 
+	// Page Params
+	pages := sites.findAllPagesByKind(KindPage)
+	for _, p := range pages {
+		if p.Path() == "sect1/regular1.en.md" {
```

Comment:
Also, when you need to add an import to get to the assert package, you by definition is breaking the standard set in the file. I think the require package is suitable in most situations. If something fails, it is bad and I want it to stop.